code/game/objects/items/devices/pizza_bomb.dm

```diff
@@ -61,7 +61,7 @@
 	if(istype(I, /obj/item/wirecutters) && primed)
 		to_chat(user, "<span class='danger'>Oh God, what wire do you cut?!</span>")
 		var/chosen_wire = input(user, "OH GOD OH GOD", "WHAT WIRE?!") in wires
-		if(!in_range(src, usr) || issilicon(usr) || !usr.canmove || usr.restrained())
+		if(!in_range(src, usr) || issilicon(usr) || HAS_TRAIT(usr, TRAIT_HANDS_BLOCKED) || usr.restrained())
```

Comment:
As an aside, this crops us a _lot_ so whatever the answer is must be applied across all of the instances of this

Reply:
I kept `restrained()` around incase anyone wanted to override it in the future. But now that i think about it, it could do with scrapping. They can just make whatever would do the restraining add the trait.